model_compression_toolkit/core/pytorch/reader/graph_builders.py

```diff
@@ -93,40 +109,50 @@ def nodes_builder(model: GraphModule,
             node_type = node.target
             if node_type == getattr:
                 node_has_activation = False
-                Logger.warning(
-                    'Pytorch model has a parameter or constant Tensor value. This can cause unexpected behaviour when '
-                    'converting the model.')
+                # technically, tensor ops like shape have no quantization issue
```

Comment:
It costs nothing to log a warning for all ops including shape, instead of manually excluding it like that without verifying it carefully.
Please remove the condition, and keep only the addition to the warning message.

Reply:
ok